internal/fnruntime/container.go

```diff
@@ -110,6 +115,25 @@ func (f *ContainerFn) Run(reader io.Reader, writer io.Writer) error {
 	return nil
 }
 
+// dockerCmdAvailable runs `docker version` to check that the docker command is
+// available, and returns an error with installation instructions if it is not.
+func dockerCmdAvailable() error {
+	const op errors.Op = "docker.check"
```

Comment:
let's remove this. This approach didn't turn out to be useful from UX perspective.

Reply:
done